code/game/objects/explosion.dm

```diff
@@ -14,6 +14,7 @@
 
 
 /proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impact_range, flash_range, adminlog = 1, ignorecap = 0, flame_range = 0 ,silent = 0)
+	set waitfor = 0

```

Comment:
I've profiled this multiple times, and this seems to be making explosions more expensive by about ~10%